encrypted-media.js

```diff
@@ -333,21 +333,20 @@
 
     'are-insecure-contexts-allowed-algorithm': { func: term_helper, fragment: 'are-insecure-contexts-allowed---deprecated', link_text: 'Are insecure contexts allowed?',  },
 
-//    'contributors': { func: contributors_helper, fragment: '', link_text: '', },
-    'keyids-initdatatype': { func: link_helper, fragment: 'format-registry/initdata/keyids.html', link_text: '"keyids" Initialization Data Format', },
+    'contributors': { func: contributors_helper, fragment: '', link_text: '', },
 
   };
 
   // These definitions referring to locations in the main EME spec are only referenced from the registry.
   var emeRegistryReferencesDefinitions = {
     'eme-spec': { func: link_helper, fragment: '#', link_text: 'Encrypted Media Extensions', },
-    'initdata-registry-cenc': { func: link_helper, fragment: '../initdata/cen.html', link_text: '"cenc" Initialization Data Format', },
-    'cenc-common-system': { func: link_helper, fragment: '../initdata/cenc.html#common-system', link_text: 'Common SystemID and PSSH Box Format', },
-    'initdata-registry-keyids': { func: link_helper, fragment: '../initdata/keyids.html', link_text: '"keyids" Initialization Data Format', },
-    'initdata-registry-webm': { func: link_helper, fragment: '../initdata/webm.html', link_text: '"webm" Initialization Data Format', },
-    'stream-registry': { func: link_helper, fragment: '../stream/', link_text: 'Encrypted Media Extensions Stream Format Registry', },
-    'stream-registry-webm': { func: link_helper, fragment: '../stream/webm.html', link_text: 'WebM Stream Format', },
-    'stream-registry-mp4': { func: link_helper, fragment: '../stream/mp4.html', link_text: 'ISO Common Encryption (\'cenc\') Protection Scheme for ISO Base Media File Format Stream Format', },
+    'initdata-registry-cenc': { func: link_helper, fragment: 'https://wwww.w3.org/TR/eme-initdata-cenc/', link_text: '"cenc" Initialization Data Format', },
```

Comment:
We should apply the same logic to use the correct URLs as you have implemented for the biblio entries. I think having these links internally consistent (ED documents link to ED documents) is much more important.